gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,42 @@
+"""Script to create release sites HT for v4 genomes."""
+import argparse
+import logging
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import dbsnp
+
+from gnomad_qc.v4.resources.annotations import get_vep
+from gnomad_qc.v4.resources.constants import CURRENT_VERSION
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("create_release_ht")
+logger.setLevel(logging.INFO)
+
+
+def remove_missing_vep_fields(vep_expr: hl.StructExpression) -> hl.StructExpression:
+    """
+    Remove fields from VEP 105 annotations that are missing in all rows.
+
+    :param vep_expr: StructExpression containing VEP 105 annotations.
+    :return: StructExpression containing VEP 105 annotations with missing fields removed.
+    """
+    vep_expr = vep_expr.drop("colocated_variants", "context")
+
+    vep_expr = vep_expr.annotate(
+        transcript_consequences=vep_expr.transcript_consequences.map(
+            lambda x: x.drop("minimised", "swissprot", "trembl", "uniparc")
+        )
```

Comment:
This is purely preference here since its a small module but I generally like to define lists like this outside of the operation as I find it easier to locate when we add/remove items.

Reply:
Hopefully next time, we will have the vep.config correct and we don't have to remove any fields.  But I can make two lists: fields_missing_in_all and fields_missing_in_transcript_consequences, still inside the function, no?

Reply:
wait, there will be a 3rd list too, like context in the first layer.

Reply:
Yeah, I went back and forth on this comment. I think having the three lists may make it less readable so lets keep it as is especially since this will only be used once